src/follow.ts

```diff
@@ -339,7 +339,6 @@ export function followPaths<S>(
 			parent.type === "CharacterClassRange" ||
 			parent.type === "ClassIntersection" ||
 			parent.type === "ClassSubtraction" ||
-			parent.type === "ExpressionCharacterClass" ||
```

Comment:
I removed this because of a TS error. I think this is due to the type of regexpp being fixed. In fact, ExpressionCharacterClass never appears here.